lib/cc/engine/scalastyle.rb

```diff
@@ -57,22 +57,28 @@ def analysis_files
       end
 
       def scalastyle_jar
-        '/home/app/scalastyle_2.11-0.6.0-batch.jar'
+        '/home/app/scalastyle_2.11-0.8.0-batch.jar'
```

Comment:
What do you think about putting `ENV SCALASTYLE_VERSION 2.11-0.8.0` in the Dockerfile? It's useful as an extracted variable there and you could read it here, meaning version bumps only need to make one change.